### ISSUE TITLE
Remove Mautic service

### ISSUE THREAD

**Issue by Brutus5000** [FAForever/faf-stack#199]
# Docker release situation

While checking the installation instructions, I noticed that Mautic 3 has been released. However, this release is not available in the official mautic docker repo. The migration manual describes that you have to update to Mautic 2.16.3 beforehand - which is neither in the mautic docker repo not available on Docker Hub. After some checking, I found that the last version 2.13 is several years old.

The Docker repository on Github is theoretically up to 2.16 (March 2020), but versions from 2.14-2.16 have also not been released. Unfortunately, the version 2.13 we are running is additionally known to have critical security vulnerabilities.

Safe operation of Mautic is no longer possible in its current state. The application must either be updated or decommissioned.

# Mautic in FAForever

Originally, we introduced Mautic because we were looking for a software to provide the emails we send with beautiful HTML templates. Ironically, up to now we still haven't achieved this goal (https://github.com/FAForever/faf-java-api/issues/322).

Mautic was then used for a few purposes:
- New users receive a welcome email with more information and can unsubscribe to a newsletter (to date we have not sent a single newsletter...).
- For updates of our privacy policy we have used Mautic for mass emails

Put simply, we use almost nothing at all of Mautic's core functionality such as landing pages, advertising campaigns, market segmentation.

On the other hand, Mautic forces us to take a further step when deleting users, as it also keeps the history of emails.

TL;DR: We don't use Mautic for what it is there for. We hardly use Mautic, but it brings us extra work. The little functionality can be implemented elsewhere.

# Summary

If the manufacturer does not manage to properly pacakge and publish Docker images, we cannot burden ourselves with this task. Especially not for a tool that we hardly use.

# Follow-up tasks
- [X] Remove Mautic from faf-stack (docker-compose.yml, config, data, scripts) (see #201)
- [x] Reconfigure faf-java-api to send mails directly to Postal
- [X] Remove Mautic support from faf-java-api (see https://github.com/FAForever/faf-java-api/pull/563)
- [ ] Delete Mautic database on production
